ingest/src/test/java/com/connexta/ingest/IngestApplicationIntegrationTest.java

```diff
@@ -252,56 +242,17 @@ public void testUnsuccessfulTransformRequest() throws Exception {
         .expect(requestTo(endpointUrlTransform))
         .andExpect(method(HttpMethod.POST))
         .andExpect(header("Accept-Version", endpointsTransformVersion))
-        .andExpect(jsonPath("$.bytes").value(TEST_FILE_SIZE))
-        .andExpect(jsonPath("$.mimeType").value(TEST_MIME_TYPE))
         .andExpect(jsonPath("$.location").value(location))
         .andRespond(withServerError());
 
     mvc.perform(
             multipart("/ingest")
                 .file("file", TEST_FILE)
-                .param("fileSize", String.valueOf(TEST_FILE_SIZE))
-                .param("fileName", "testFile.txt")
-                .param("title", "qualityTitle")
-                .param("mimeType", TEST_MIME_TYPE)
```

Comment:
These extra params should be removed in the other ingest requests as well (see the `testStoreRequest` methods).